docs/preview/actions-runner-controller-2/README.md

```diff
@@ -0,0 +1,142 @@
+# Autoscaling Runner Scale Sets mode
+
+**⚠️ This mode is currently only available for a limited number of organizations.**
+
+This new autoscaling mode brings numerous enhancements (described in the following sections) that will make your experience more reliable and secure.
+
+## How it works
+
+![arc_hld_v1 drawio (1)](https://user-images.githubusercontent.com/568794/212665433-2d1f3d6e-0ba8-4f02-9d1b-27d00c49abd1.png)
+
+In addition to the increased reliability of the automatic scaling, we have worked on these improvements:
+
+- No longer require cert-manager as a prerequisite for installing actions-runner-controller
+- Reliable scale-up based on job demands and scale-down to zero runner pods
+- Reduce API requests to `api.github.com`, no more API rate-limiting problems
+- The GitHub Personal Access Token (PAT) or the GitHub App installation token is no longer passed to the runner pod for runner registration
+- Maximum flexibility for customizing your runner pod template
+
+### Demo
+
+https://user-images.githubusercontent.com/568794/212668313-8946ddc5-60c1-461f-a73e-27f5e8c75720.mp4
+
+## Setup
+
+### Prerequisites
+
+1. Create a K8s cluster, if not available.
+    - If you don't have a K8s cluster, you can install a local environment using minikube. See [installing minikube](https://minikube.sigs.k8s.io/docs/start/).
+1. Install helm 3, if not available. See [installing Helm](https://helm.sh/docs/intro/install/).
+
+### Install actions-runner-controller
+
+1. Install actions-runner-controller using helm 3. For additional configuration options, see [values.yaml](https://github.com/actions/actions-runner-controller/blob/master/charts/actions-runner-controller-2/values.yaml)
+
+    ```bash
+    NAMESPACE="arc-systems"
+    helm install arc \
+        --namespace "${NAMESPACE}" \
+        --create-namespace \
+        oci://ghcr.io/actions/actions-runner-controller-charts/actions-runner-controller-2 \
+        --version 0.1.0
+    ```
+
+1. Generate a Personal Access Token (PAT) or create and install a GitHub App. See [Creating a personal access token](https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token) and [Creating a GitHub App](https://docs.github.com/en/developers/apps/creating-a-github-app).
+    - For the list of required permissions, see [Authenticating to the GitHub API](https://github.com/actions/actions-runner-controller/blob/master/docs/authenticating-to-the-github-api.md#authenticating-to-the-github-api).
+
+1. You're ready to install the autoscaling runner set. For additional configuration options, see [values.yaml](https://github.com/actions/actions-runner-controller/blob/master/charts/auto-scaling-runner-set/values.yaml)
+    - **Choose your installation name carefully**, you will use it as the value of `runs-on` in your workflow.
+
+    ```bash
+    # Using a Personal Access Token (PAT)
+    INSTALLATION_NAME="arc-runner-set" 
+    NAMESPACE="arc-systems"
```

Comment:
we might want to suggest customers deploy the autoscalingrunnerset into a different namespace to show best security practices.